src/reyna/Reyna/Http/ReynaWebRequest.cs

```diff
@@ -70,6 +76,7 @@ public Result Send(string content)
             {
                 var response = webException.Response as HttpWebResponse;
                 statusCode = this.GetStatusCode(response);
+                Logger.Error("ReynaWebRequest.Send Error {0} Status code {1}", webException.ToString(), statusCode);
```

Comment:
better to log exception object instead of webException.ToString() to get full stack
@Youhana-Hana